app/models/course.rb

```diff
@@ -1,12 +1,20 @@
 class Course < ActiveRecord::Base
+
+  # Model Definitions
   enum semester: [:winter, :summer, :fall]
   validates :prefix, presence: true, format: { with: /\A[A-Z]+\z/, 
   	message: "needs uppercase letters" }
-  validates :code, presence: true, numericality: true
-  validates :name, format: { with: /\A\w+\z/, message: "not a proper name" }
+  validates :code, presence: true, inclusion: { in: 100..9999 }, numericality: true
+  validates :name, format: { with: /\A[\w\s]+\z/, message: "not a proper name" }
```

Comment:
Yeah that makes sense, I was surprised when I was unable to create a course in the Console.